private-templates-service/adaptivecards-templating-service/src/authproviders/AzureADProvider.ts

```diff
@@ -44,8 +44,11 @@ export class AzureADProvider implements AuthenticationProvider {
       }
     }
 
+    // Check expiry date on token
+    result = new Date() <= new Date(decodedToken.payload.exp * 1000); 
+
     // Check aud of token matches the client ID of env app
-    result = "#{CLIENT_ID_TOKEN}#" === decodedToken.payload.aud;
+    result = "#{CLIENT_ID_TOKEN}#" === decodedToken.payload.appid || "#{CLIENT_ID_TOKEN}#" === decodedToken.payload.aud;
```

Comment:
are we going to have to change both of these now to make the app run locally?

Reply:
Moved to client id token to variable so it only needs to be changed once